pages/[lng]/blog/[slug].tsx

```diff
@@ -138,9 +145,13 @@ const BlogSlug: FC<any> = ({
 
 export const getServerSideProps: GetServerSideProps = async ({
   params,
+  res,
   req,
 }) => {
-  const brand = await useBrand(req);
+  const [brand, ] = await Promise.all([
```

Comment:
const headerImage dimasukin kesini jg yaa
```suggestion
  const [
    brand,
    headerImage
  ] = await Promise.all([
    useBrandCommon(req, params), 
    getBlogHeaderImage(GRAPHQL_URI(req)),
    useAuthToken({req, res, env: process.env})
  ])
```